bin/yb-docker-ctl

```diff
@@ -418,11 +418,15 @@ class YBDockerControl():
                         init_db_cmd = "{} {}/initdb -U postgres -D {}".format(
                             init_db_envs, pg_bin_dir, tmp_data_dir)
                         # Setup the list based cmdline.
-                        print("Running initdb to initialize PostgreSQL metadata "
-                              "in the YugaByte cluster. This may take up to a minute.")
                         docker_cmd = ['docker', 'exec', '-it', self.YB_TSERVER_FORMAT.format(1)]
                         docker_cmd.extend(["bash", "-c", init_db_cmd])
+                        print(' '.join(docker_cmd))
```

Comment:
Do we really want to print out the whole command?

Reply:
Yes, I want to keep it consistent with how we print out commands for yb-master, yb-tserver. Also, for now, provides more transparency to users.